src/Ombi.Core/Engine/RequestEngineResult.cs

```diff
@@ -1,11 +1,31 @@
-﻿namespace Ombi.Core.Engine
+﻿using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+
+namespace Ombi.Core.Engine
 {
     public class RequestEngineResult
     {
         public bool Result { get; set; }
         public string Message { get; set; }
         public bool IsError => !string.IsNullOrEmpty(ErrorMessage);
         public string ErrorMessage { get; set; }
+        public ErrorCode? ErrorCode { get; set; }
         public int RequestId { get; set; }
     }
+
+    [JsonConverter(typeof(StringEnumConverter))]
+    public enum ErrorCode {
+        AlreadyRequested,
```

Comment:
Can we please assign the explicit values to these, so we don't break things on reordering for example

Reply:
StringEnumConverter will return the name of the member, not an ID. Reordering won't break this.
![image](https://user-images.githubusercontent.com/34862846/139841329-523020fd-8233-4f91-90da-95262b96111b.png)

Reply:
Ok! Didn't spot that :)